.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ name: Release Artifacts
 jobs:
   build-linux:
     name: "Build Linux"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Is there a reason why this does not work with latest?

Reply:
Na verdade funciona com versão mais nova, porem é recomendável sempre fazer builder do appimage com versões mais antigas do ubuntu para que assim ele seja suportado por um número maior de destruições linux